style.css

```diff
@@ -0,0 +1,82 @@
+body {
+    font-family: 'Times New Roman', Times, serif;
+    font-size: 1.6rem;
+    line-height: 1.5;
+    text-align: center;
+    color: blueviolet;
+    margin: 0;
+}
+
+h1 {
+    font-size: 4rem;
+    margin-bottom: 0;
+}
+
+/* h3 {
+  font-size: 3rem;
+  margin-bottom: 0;
+} */
```

Comment:
- _(optional)_ Also I recommend removing unnecessary commented code. Make sure to keep your code clean

Reply:
Will do @m05e5 Thank you!